guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_approach.md

```diff
@@ -34,6 +33,10 @@ To extend the parent theme's styles in your theme:
 
 Extending a theme using <code>_extend.less</code> is the simplest option when you are happy with everything the parent theme has, but want to add more styles.
 
+<div class="bs-callout bs-callout-info" id="info-extend-priority">
```

Comment:
We don't use HTML like this for callouts, or notes. Check out our Contributor guide for info on how to format callouts: https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/basic_template.html.

Reply:
Also, use markdown ALWAYS when you can.